test/kessel_test.go

```diff
@@ -0,0 +1,191 @@
+package test
+
+import (
+	"context"
+	"fmt"
+	"github.com/go-kratos/kratos/v2/log"
+	"github.com/go-kratos/kratos/v2/middleware/tracing"
+	v0 "github.com/project-kessel/relations-api/api/relations/v0"
+	"github.com/stretchr/testify/assert"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+	"os"
+	"testing"
+)
+
+var localKesselContainer *LocalKesselContainer
+
+func TestMain(m *testing.M) {
+	var err error
+	logger := log.With(log.NewStdLogger(os.Stdout),
+		"ts", log.DefaultTimestamp,
+		"caller", log.DefaultCaller,
+		"trace.id", tracing.TraceID(),
+		"span.id", tracing.SpanID(),
+	)
+
+	localKesselContainer, err = CreateKesselAPIContainer(logger)
+	if err != nil {
+		fmt.Printf("Error initializing Docker localKesselContainer: %s", err)
+		os.Exit(-1)
```

Comment:
Curious to understand why "-1" exit code and not just a non zero exit code?

Reply:
Good Questions

Go's os.Exit function only accepts values in the range of 0 to 255 as the exit code. Negative values are converted to positive ones in a modular fashion, usually modulo 256. 

os.Exit(-1) would give:
`Process finished with the exit code 255.`

I used it because I see it is already used in spicedb_test.go and relationship_test.go

Reply:
thanks Abhishek